keycloak/keycloak-admin-ui
# Fix default values for group and role mappers

## Motivation
Fixes https://github.com/keycloak/keycloak-admin-ui/issues/1479.

## Brief Description
For both the group-ldap-mapper and role-ldap-mapper user fed LDAP mappers, if the vendor of the LDAP provider is Active Directory, then there should be different default values for Group/Role Object Classes and Membership User LDAP Attribute fields. The other four vendors share the same default values for those two fields. See https://github.com/keycloak/keycloak-admin-ui/issues/1479 for more info.

## Verification Steps
1. Create a User Federation LDAP provider and set the vendor to Active Directory.
2. Go to User Federation > LDAP > LDAP Mappers.
3. Create a new mapper.
4. Choose **group-ldap-mapper** as the mapper type.
5. Verify the following default values: 
    - Group Object Classes: `group`
    - Membership User LDAP Attribute: `cn`
6. Using the same LDAP provider, create a new mapper and choose **role-ldap-mapper** as the mapper type.
7. Verify the following default values: 
    - Role Object Classes: `group`
    - Membership User LDAP Attribute: `cn`
8. Create a new User Federation LDAP provider and set the vendor to any vendor other than Active Directory.
9. Go to User Federation > LDAP > LDAP Mappers.
10. Create a new mapper.
11. Choose **group-ldap-mapper** as the mapper type.
12. Verify the following default values: 
    - Group Object Classes: `groupOfNames`
    - Membership User LDAP Attribute: `uid`
13. Using the same LDAP provider, create a new mapper and choose **role-ldap-mapper** as the mapper type.
14. Verify the following default values: 
    - Role Object Classes: `groupOfNames`
    - Membership User LDAP Attribute: `uid`

## Checklist:

## Additional Notes
None
